cmd/anonymizer/app/flags.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var QueryGRPCPort, MaxSpansCount int
+var QueryGRPCHost, TraceID, OutputDir string
+var HashStandardTags, HashCustomTags, HashLogs, HashProcess bool
+
+const (
+	queryGRPCHostFlag    = "query.host"
+	queryGRPCPortFlag    = "query.port"
+	outputDirFlag        = "output.dir"
+	traceIDFlag          = "trace.id"
+	hashStandardTagsFlag = "hash.standardtags"
+	hashCustomTagsFlag   = "hash.customtags"
+	hashLogsFlag         = "hash.logs"
+	hashProcessFlag      = "hash.process"
+	maxSpansCount        = "max.spanscount"
```

Comment:
I would recommend using `-` as separator between words. And split all words, e.g. `standardtags` -> `standard-tags`